drivers/vmwarevsphere/cloudinit.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/vmware/govmomi/object"
 	"github.com/vmware/govmomi/vim25/soap"
 	"github.com/vmware/govmomi/vim25/types"
-	"gopkg.in/yaml.v2"
+	"sigs.k8s.io/yaml"
```

Comment:
@luthermonson might want to double check this won't cause issues. The sig one reads json tags, this may affect vsphere.